www/js/diary/infinite_scroll_list.js

```diff
@@ -386,31 +424,33 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
             $scope.$apply(() => {
                 tripgj.start_display_name = startName;
                 tripgj.end_display_name = endName;
+                if (tripgj.confirmed_place) {
+                  tripgj.confirmed_place.display_name = endName;
+                }
             });
         });
     }
 
-    const fillTrajectoriesForTripAsync = function(trip) {
-        mapLimiter.schedule(() => Timeline.confirmedTrip2Geojson(trip)).then((tripgj) => {
```

Comment:
We are still instantiating `mapLimiter` for this component, do we not need to use it during `fillTrajectoriesForTrip` anymore? If so then we may consider removing mapLimiter from above

Reply:
Yes, I think we can remove `mapLimiter` altogether

Reply:
removed in 6e49ea91ecafa761a47df38703c4125536424119